frontend/src/host_orchestrator/constants_arm64.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build arm64
+// +build arm64
+
+package main
+
+const (
+	defaultCVDBinAndroidBuildID     = "11125998"
```

Comment:
Be careful when choosing a build id that is significantly different for both platforms:
There have been several changes introduced into `cvd` recently, we've tried to keep up from the HO side, but it's inevitable to fall behind sometimes. If you choose different versions of `cvd` for different architectures the code that calls it may be incompatible with at least one of them.

Reply:
QQ: Does upgrading cvd binary build ID for x86 cause any problem? If there's no issue, I want to upgrade it as similar number which arm64 have, though it couldn't be exact same number.

Reply:
10797123 is the most similar build id for arm64

Reply:
Then matching or downgrading arm64 version similar to x86_64 looks better? No strong opinions from me.

Reply:
As Jorge mentioned, cvd has been changed a lot recently, so for the stability, using current version for both x86 and arm64 looks good to me, and then, we can update them soon after checking stability and compatibility with HO

Reply:
That makes sense. I'll create a short patch that downgrading arm64 build ID.

Reply:
https://github.com/google/android-cuttlefish/pull/386

Reply:
Merged following PR, so set as resolved :)